src/common/debug.c

```diff
@@ -263,3 +264,4 @@ daos_key2str(daos_key_t *key)
 	thread_key_buf_idx = (thread_key_buf_idx + 1) % DF_KEY_MAX;
 	return buf;
 }
+#endif
```

Comment:
(style)  adding a line without newline at end of file

Reply:
This should be easy to fix.

Reply:
Fixed